pkg/services/user/userimpl/user.go

```diff
@@ -72,6 +72,14 @@ func (s *Service) GetUsageStats(ctx context.Context) map[string]interface{} {
 	}
 
 	stats["stats.case_insensitive_login.count"] = caseInsensitiveLoginVal
+
+	count, err := s.store.CountUserAccountsWithEmptyRole(ctx)
```

Comment:
hmm I would perhaps not make a method on top of the sqlstore.

I would put this in a function if possible?

Just make the sql outside as part of a function and then call it from here.

Saying this to not pollute the sqlstore.

Reply:
I don't think I understand correctly. Please elaborate and maybe provide an example that I can use as reference 🙏

Reply:
I think this is fine. We should not perform sql outside the store so lets keep it there.